kv/union_store.go

```diff
@@ -39,6 +39,22 @@ type UnionStore interface {
 	GetMemBuffer() MemBuffer
 }
 
+// AssertionType is the type of a assertion.
+type AssertionType int
+
+// The AssertionType constants.
+const (
+	NoCheck AssertionType = iota
```

Comment:
We can use the type in kvproto directly.

Reply:
em, seems not a good idea to let package `kv` have dependency on `kvproto`.

Reply:
I agree with @disksing

Reply:
At least we can make the name consistent, use `None/Exist/NotExist`.